ufo2fontir/src/source.rs

```diff
@@ -290,7 +299,73 @@ impl DesignSpaceIrSource {
 struct StaticMetadataWork {
     designspace_file: PathBuf,
     designspace: Arc<DesignSpaceDocument>,
-    glyph_order: Arc<Vec<String>>,
+    glyph_names: Arc<HashSet<String>>,
+}
+
+fn default_master(designspace: &DesignSpaceDocument) -> Option<&designspace::Source> {
+    // The master at the default location on all axes
+    // TODO: fix me; ref https://github.com/googlefonts/fontmake-rs/issues/22
+    warn!("Using an incorrect algorithm to determine the default master");
+    let default_location: DesignSpaceLocation = designspace
+        .axes
+        .iter()
+        .map(|a| (a.name.clone(), OrderedFloat::from(a.default)))
+        .collect();
+    designspace
+        .sources
+        .iter()
+        .find(|source| to_ir_location(&source.location) == default_location)
+}
+
+fn load_lib_plist(ufo_dir: &Path) -> Result<plist::Dictionary, WorkError> {
+    let lib_plist_file = ufo_dir.join("lib.plist");
+    if !lib_plist_file.is_file() {
```

Comment:
you should not expecet lib.plist to be present, the whole file is optional. If missing return an  empty plist dict

Reply:
Tricky bit is if one appears we would want to trigger a recompile. Tracking in https://github.com/googlefonts/fontmake-rs/issues/49.